pkg/features/kube_features.go

```diff
@@ -208,7 +208,8 @@ const (
 	CRIContainerLogRotation featuregate.Feature = "CRIContainerLogRotation"
 
 	// owner: @krmayankk
-	// beta: v1.14
+	// beta:  v1.14
+	// ga:    v1.22
```

Comment:
Can we get this in v1.21?

Reply:
the KEP is actually assigned to 1.21: https://github.com/kubernetes/enhancements/issues/213

Reply:
correct, do i just need to fix this item here to say `ga: 1.21`  ? Who owns reviewing this ?

Reply:
Yes, it should say 1.21. Any feature approver can review this file.

Reply:
fixed